peps/pep-0667.rst

```diff
@@ -3,13 +3,15 @@ Title: Consistent views of namespaces
 Author: Mark Shannon <mark@hotpy.org>,
         Tian Gao <gaogaotiantian@hotmail.com>
 Discussions-To: https://discuss.python.org/t/46631
-Status: Accepted
+Status: Final
 Type: Standards Track
 Created: 30-Jul-2021
 Python-Version: 3.13
 Post-History: 20-Aug-2021, 22-Feb-2024
 Resolution: https://discuss.python.org/t/pep-667-consistent-views-of-namespaces/46631/25
 
+.. canonical-doc:: :external+py3.13:func:`locals`
```

Comment:
With the PEP status resolved, that just leaves the question of whether or not we should expand on the canonical docs reference (since the PEP affected multiple things that are documented in different places).

Reply:
We can always come back and add more links later if we decide it is necessary (although I doubt it will be necessary).